cfgov/tccp/jinja2/tccp/landing_page.html

```diff
@@ -40,9 +52,9 @@ <h1>{{ heading }}</h1>
             </div>
 
             <p>
-                If you don’t know your credit score,
-                <a href="/ask-cfpb/where-can-i-get-my-credit-scores-en-316/">find out for free</a>
-                or play around to find out how credit score affects rates.
+                If you don’t know it,
+                <a href="/ask-cfpb/where-can-i-get-my-credit-scores-en-316/"
+                    >find your credit score</a>.
             </p>
```

Comment:
Looks like in the latest mockups this line is set as [helper text](https://cfpb.github.io/design-system/components/helper-text) and is ~ 5 px under the dropdown (so we can probably pull the `p` into the `div.o-form_group`

Reply:
Great suggestion, just pushed a commit that does this!